tests/test_cli/test_fetch.py

```diff
@@ -125,6 +131,11 @@ def test_fetch_positive(self, *mocks):
             [*CLI_LOG_OPTION, "fetch", "--local", "author/name:0.1.0"],
             standalone_mode=False,
         )
+        self.runner.invoke(
+            cli,
+            [*CLI_LOG_OPTION, "fetch", "--remote", "author/name:0.1.0"],
+            standalone_mode=False,
+        )
```

Comment:
fetching repeatedly to same folder...should cause clashes

Reply:
OK, will split in 3 tests